app/controllers/pages_controller.rb

```diff
@@ -1,7 +1,25 @@
 class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [:home, :about]
 
-  def home; end
+  def home
+    @policy_making = PolicyMaking.new
```

Comment:
I'm curious: why making a new PolicyMaking here and why assign it the topic and country later on? This @policy_making seems to only be relevant when there's an error of some sort, but I don't fully understand why we need it sorry.
Note: what about moving the PolicyMaking.new to after the if params[:policy_making].present? so that nothing really happens when no params are there?